components/config-util/src/main/java/com/wso2telco/core/config/model/MobileConnectConfig.java

```diff
@@ -3098,6 +3099,15 @@ public String getGrantTypes() {
         public void setGrantTypes(String grantTypes) {
             this.grantTypes = grantTypes;
         }
+
+        @XmlElement(name = "MIGUrl")
```

Comment:
cannot we use the host name from carbon?

Reply:
let me have a look.

Reply:
This is fixed now. Please check.